core/src/utilities/helpers/navigation-helpers.js

```diff
@@ -62,8 +62,11 @@ const orderNodes = nodes => {
   });
 };
 
-export const groupNodesBy = (nodes, property) => {
+export const groupNodesBy = (nodes, property, useVirtualGroups) => {
```

Comment:
Is it necessary to add `useVirtualGroups` parameter if it is only used in one file which uses that flag with `true` anyways?

Reply:
We are not sure yet if we provide a setting for it or not. I f yes, we want to have the "all-ungrouped-nodes-on-top" functionality there as well.